packages/gcm/lib/apk/config.yaml

```diff
@@ -0,0 +1,13 @@
+services:
+  - name: gcm::GcmService
```

Comment:
This is the name of the service as given in the gcm.mojom file (see https://github.com/flutter/engine/blob/master/sky/services/gcm/gcm.mojom ). This is a bit redundant. If we could annotate the mojom file directly, we could probably avoid this section.